submissions/zadorogniyandrey/action-menu/js/main.js

```diff
@@ -0,0 +1,71 @@
+  },
+  {
+    id: 4,
+    title: "Accenture",
+    image: "img/4.jpeg.webp",
+    description:
+      'Accenture была создана как технологическое подразделение бухгалтерской фирмы под названием "Arthur Andersen", но в 1989 году она превратилась в независимую компанию. Сегодня это одно из самых крупных предприятий работающих в сфере цифрового маркетинга и предоставляющих бизнес-консультирование, технологические решения, приложения для безопасности и повышения продуктивности и т.д. Но компания не собирается останавливаться на этом. Следующий приоритетный и довольно амбициозный шаг для Accenture – квантовые вычисления. Сегодня компания обслуживает клиентов из более чем 120 стран. Клиентская база Accenture включает даже таких гигантов как Microsoft, Oracle и SAP. И да, если вы не в курсе, название "Accenture" происходит от фразы "акцент на будущем" (the accent of the future).',
+  },
+  {
+    id: 5,
+    title: "Cisco Systems",
+    image: "img/5.jpeg.webp",
+    description:
+      "Cisco Systems — пятая самая успешная IT компания в мире по доходу. Ее годовая прибыль уже превысила $52 млрд. Обычно Cisco знают как производителя коммутаторов, роутеров и другого сетевого оборудования. Но с 2015 года компания решила углубиться в инновационные сферы, связанные с ПО. Сегодня Cisco также занимается интернетом вещей, кибербезопасностью, облачными вычислениями, дата-центрами и анализом данных. Компания была основана в 1984 году Леонардом Босаком и Сандрой Лернер. Они создали коммерчески успешный многопротокольный маршрутизатор, который соединял ранее несовместимые компьютеры и был построен на базе стэндфордского Blue Box Router. Интересно, что в 2000 году Cisco была самой дорогой компанией в мире, благодаря так называемому Пузырю доткомов (Dot-com bubble). Рыночная капитализация компании достигла рекордной отметки в $500 млрд. Начиная с 1993 года Cisco поглотила примерно 170 разных предприятий. Сегодня она имеет представительства в более чем 160 странах, а главный офис находится в Сан-Хосе, Калифорния, США.",
+  },
+];
+const menu = document.querySelector(".sidebar__menu");
+const main = document.querySelector(".main");
+const menuItems = (item, menuList) => {
+  item.map(function (item) {
+    let sideWrap = document.createElement('div');
+    sideWrap.innerHTML = (`<li class="sidebar__menu__item"><button class="sidebar__btn" data-company = ${item.id}>${item.title}</button></li>`);
+    menuList.append(sideWrap)
+  });
+};
+const content = (i = 1, arr = companies) => {
```

Comment:
Functions denote actions hence their names should start with a verb. Variables containing strings, numbers, and objects are normally nouns. Boolean variables/functions start with is, does, has etc. A variable containing multiple entities and functions returning lists containing entity names in plural form.

Reply:
> Thank you! I didn't know about `create fragment`. Is creating `templates` the right solution?

Read carefully about the `template` tag. [Link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template).
In your case, `document.createFragment` will be enough.